### PR TITLE
Collect and expose memory cgroup stats

### PR DESCRIPTION
**Pull request by CyberDem0n**

### CODE REVIEW

File: system_stats.c [hunk 1]
```diff
@@ -178,4 +235,12 @@ void system_stats_init(void)
 	struct utsname un;
 	if (uname(&un) == 0)
 		sysname = pstrdup(un.release);
+
+	if (memory_cgroup_mount != NULL) {
+		const char prefix[] = "/memory.";
+		memory_cgroup_len = strlen(memory_cgroup_mount);
+		memory_cgroup = repalloc(memory_cgroup_mount, memory_cgroup_len + 23);
```
Comment (a1exsh) [inline system_stats.c]: Why 23?

Comment (CyberDem0n) [reply system_stats.c]:
```bash
$ echo "/memory.usage_in_bytes" | wc -c
23
```

Comment (a1exsh) [reply system_stats.c]: Yeah, obviously, but the point is that it deserves a code comment.

It would also help to reflect the fact that `memory_cgroup` is a filename (as opposed to `cgroup_memory`).  Otherwise it's pretty confusing.

File: system_stats.c [hunk 2]
```diff
@@ -108,7 +166,7 @@ static system_stat read_proc_stat(void)
 		return st;
 
 	while (fgets(buf, sizeof(buf), stfd))
-		if (sscanf(buf, "%s %llu", param, &value) == 2) {
```
Comment (a1exsh) [inline system_stats.c:111]: Unrelated change ;)

**Comment by a1exsh** [comment]
:+1:
